### DISCUSSION TITLE
Problems calculating absolute delay for each date using PyAPS

### DISCUSSION THREAD

**Discussion by anaferreira97** [insarlab/MintPy#779] [Q&A] [answered]
Hello everyone!

I'm trying to run Mintpy with GMTSAR processor but I'm having problems calculating absolute delay for each date using PyAPS. I'm getting the error "TypeError: Can't broadcast (243, 227) -> (1, 3970, 2440)". The entire log is attached.
I'm trying to understand the error but without success... Can you please help me?

[mintpy_log.txt](https://github.com/insarlab/MintPy/files/8761663/mintpy_log.txt)

**Comment by yunjunz** [comment, marked as answer]
Based on your attached log file, the error is due to the size of the DEM (243 x 277), which is different from the input interferograms' size (3970 x 2440).

**Reply by anaferreira97** [reply]
I already tried a DEM bigger than the input interferograms size but the error still appears.

**Reply by yunjunz** [reply]
You could plot the DEM and the unwrapped interferograms to check if the DEM covers the entire region of the interferogram.

What I meant was that it seems like the DEM you loaded into mintpy is in much lower resolution (~16 x 8 times) than the interferograms. We would want both files are in the exact spatial extent/coverage and spatial resolution.

**Reply by anaferreira97** [reply]
All dataset have the same settings and I still have the error... The coordinate system, extent, resolution, dimensions and pixel size... Everything is the same! I can't understand what is wrong.

**Reply by yunjunz** [reply]
Hmm... Could you show the latest print out msg  for the error?

**Reply by anaferreira97** [reply]
I was testing in the wrong dataset, thank you so much for your help!
